release-schema.json

```
@@ -182,6 +214,66 @@
           "title": "Buyer",
           "description": "The organization aiming to conclude a contract with a supplier or to use the goods, works or services in this lot. This may be different from the procuring entity who may be specified in the tender data.",
           "$ref": "#/definitions/OrganizationReference"
+        },
+        "additionalClassifications": {
+          "title": "Additional classifications",
+          "description": "An array of additional classifications for this lot.",
```

The standard's wording is against style guide.

```suggestion
          "description": "Additional classifications for this lot.",
```